packages/core/src/components/SupportButton/SupportButton.tsx

```diff
@@ -120,6 +126,7 @@ export const SupportButton = ({ children }: PropsWithChildren<Props>) => {
         onClose={popoverCloseHandler}
       >
         <List className={classes.popoverList}>
+          {title && <ListItem alignItems="flex-start">{title}</ListItem>}
```

Comment:
Does this warrant a different text style? Maybe a `<Typography variant="something something"` from here? https://material-ui.com/components/typography/